admin/about_me.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+require 'functions.php';
+
+$db = connectDatabase();
+$stmt = $db->query("SELECT `bio_title`,`bio`,`contact_title`,`email`,`telephone` FROM `about_me`;");
+$about_me_data = $stmt->fetch();
+
+?>
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+</head>
+<body>
+
+    <form method="POST" action="submit_about_me.php">
+        <p>Bio title</p><input type="text" name="bio_title" value="<?php echo $about_me_data["bio_title"] ?>">
```

Comment:
use `label`s instead of `p` tags and put the inputs on their own lines, as uv exceeded the maximum line length